core/templates/dev/head/pages/exploration_player/ProgressDotsDirective.js

```diff
@@ -63,9 +63,9 @@ oppia.directive('progressDots', [
           });
 
           $scope.changeActiveDot = function(index) {
-            playerPositionService.setActiveCardIndex(index);
+            PlayerPositionService.setActiveCardIndex(index);
             $rootScope.$broadcast('updateActiveStateIfInEditor',
-                                  playerPositionService.getCurrentStateName());
+                                  PlayerPositionService.getCurrentStateName());
```

Comment:
No need to indent arguments from the position of the parenthesis, just indenting it once is enough 
![indent3](https://user-images.githubusercontent.com/7807557/31108247-9436026e-a815-11e7-8771-e34bb1ba744a.jpeg)